### ISSUE TITLE
can not execute fabmanager create-admin –app caravel

### ISSUE THREAD

**Issue by qinzl1** [airbnb/caravel#711]
Traceback (most recent call last):
  File "/usr/local/bin/fabmanager", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/usr/local/lib/python2.7/site-packages/distribute-0.6.14-py2.7.egg/pkg_resources.py", line 2671, in <module>
    working_set.require(__requires__)
  File "/usr/local/lib/python2.7/site-packages/distribute-0.6.14-py2.7.egg/pkg_resources.py", line 654, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/local/lib/python2.7/site-packages/distribute-0.6.14-py2.7.egg/pkg_resources.py", line 552, in resolve
    raise DistributionNotFound(req)
pkg_resources.DistributionNotFound: Flask-WTF>=0.9.1

**Comment by bobcolner** [comment]
Also unable to execute fabmanager create-admin. 

λ AirBob Dropbox → fabmanager create-admin --app caravel
Username [admin]: admin
User first name [admin]: admin
User last name [user]: user
Email [admin@fab.org]: admin@fab.org
Password:
Repeat for confirmation:
/usr/local/lib/python3.5/site-packages/flask/exthook.py:71: ExtDeprecationWarning: Importing flask.ext.script is deprecated, use flask_script instead.
  .format(x=modname), ExtDeprecationWarning
Was unable to import caravel Error: make_declarative_base() takes 1 positional argument but 2 were given

**Comment by xrmx** [comment]
@qinzl1 it looks like you are missing dependencies or they are outdated. *pip install --upgrade Flask-WTF* should do the trick

@bobcolner completely different issue than the reporter, please don't hijack issues

**Comment by bobcolner** [comment]
@xrmx: just wow.

**Comment by qinzl1** [comment]
i upgrade all about flask ,also can not create admin 
a new issues
/usr/local/lib/python2.7/site-packages/flask/exthook.py:71: ExtDeprecationWarning: Importing flask.ext.sqlalchemy is deprecated, use flask_sqlalchemy instead.
  .format(x=modname), ExtDeprecationWarning
/usr/local/lib/python2.7/site-packages/flask/exthook.py:71: ExtDeprecationWarning: Importing flask.ext.sqlalchemy._compat is deprecated, use flask_sqlalchemy._compat instead.
  .format(x=modname), ExtDeprecationWarning
Username [admin]: admin
User first name [admin]: 
User last name [user]: 
Email [admin@fab.org]: 
Password: 
Repeat for confirmation: 
/usr/local/lib/python2.7/site-packages/flask_cache/__init__.py:152: UserWarning: Flask-Cache: CACHE_TYPE is set to null, caching is effectively disabled.
  warnings.warn("Flask-Cache: CACHE_TYPE is set to null, "
/usr/local/lib/python2.7/site-packages/flask/exthook.py:71: ExtDeprecationWarning: Importing flask.ext.cache is deprecated, use flask_cache instead.
  .format(x=modname), ExtDeprecationWarning
2016-07-03 20:07:52,829:ERROR:flask_appbuilder.security.sqla.manager:DB Creation and initialization failed: No module named pysqlite2

**Comment by xrmx** [comment]
@qinzl1 you have done something wrong installing caravel since you are missing a lot of fab dependencies. Also the errors are in plain english so just keep installing the missing pieces.

**Comment by xrmx** [comment]
No more reporter feedback in 1 month, closing
